packages/ui/src/components/collectibles/native/collectible-text.native.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+
+// import { sanitize } from 'dompurify';
+import { Text } from '../../../../native';
+import { CollectibleCardLayout } from './collectible-card-layout.native';
+
+interface CollectibleTextProps {
+  src: string;
+}
+
+export function CollectibleText({ src }: CollectibleTextProps) {
+  const [content, setContent] = useState<string | null>(null);
+
+  useEffect(() => {
+    const fetchContent = async () => {
+      try {
+        const response = await fetch(src);
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+        const data = await response.json();
+        setContent(JSON.stringify(data, null, 2));
+      } catch (err) {}
+    };
+
+    void fetchContent();
+  }, [src]);
+
+  return (
+    <CollectibleCardLayout bg="ink.text-primary" p="4">
+      <Text color="ink.background-secondary" textAlign="left">
+        {/* FIXME 
+        - implement alternative for dompurify in native
+        - I tried using jsdom as a polyfill but then hit an error Can't resolve 'vm' in jsdom
+        - maybe we should write our own sanitizer?
+        */}
+        {/* {content ? sanitize(content) : ''} */}
```

Comment:
I tried using `dompurify.sanitize` here but it seems not to work on native. 

I tried a polyfill of `jsdom` but then that failed and needed another `polyfill` so I need to investigate a better way. 

Maybe we could run `sanitize` in the query instead? So that we just provide the data ready to render